lib/slide.dart

```diff
@@ -73,25 +105,80 @@ class Slide extends StatelessWidget {
   }
 }
 
-// TODO: update this to match the real design when it gets complete.
 List<Slide> theSlides(BuildContext context) {
   final lang = AppLocalizations.of(context);
-  const theSameAsset = AssetImage('assets/ubuntu-on-wsl.png');
+  final bodyStyle = _bodyTextStyle(context);
+  final linkStyle = bodyStyle.copyWith(
+    color: Colors.white,
+    decoration: TextDecoration.underline,
+  );
   return [
     Slide(
-      image: theSameAsset,
+      image: SvgPicture.asset('assets/1-Ubuntu on WSL.svg'),
+      title: lang.welcome,
```

Comment:
lang.welcome needs updating? it's still "Welcome to Ubuntu"

Reply:
Good catch! It was supposed to be "Welcome to Ubuntu WSL". Thanks. I'll get those issues fixed.